dask/dataframe/io/parquet/arrow.py

```diff
@@ -40,59 +40,52 @@ def _determine_dataset_parts(fs, paths, gather_statistics, **kwargs):
     dataset.
     """
     parts = []
+    dataset_kwargs = kwargs.get("dataset", {})
```

Comment:
Any reason not to make this a regular keyword `dataset=None`?

Reply:
Actually, it looks like `**kwargs` isn't used for any other purpose, so I would strongly prefer to just accept `dataset_kwargs=None` and remove kwargs, unless I'm missing something.

Reply:
Agreed - `kwargs` is only used for "dataset"-related calls in this particular case.  In an original version of the code/logic, the same kwargs were used for both "read" and "dataset"-related calls, but this can be cleaned up a bit now.